gtdynamics/optimizer/Optimizer.cpp

```diff
@@ -21,9 +23,41 @@ using gtsam::Values;
 
 Values Optimizer::optimize(const NonlinearFactorGraph& graph,
                            const Values& initial_values) const {
+  std::cout<< "running LM\n";
```

Comment:
Remove in final PR. Why do we still have this optimize ?

Reply:
done. We keep this optimize for compatibility of existing code and test.